guide/src/class.md

```diff
@@ -350,7 +354,7 @@ impl SubClass {
 
 ## Object properties
 
-PyO3 supports two ways to add properties to your `#[pyclass]`:
+PyO3 supports two ways to add properties to your `#[pyclass]` struct:
 - For simple fields with no side effects, a `#[pyo3(get, set)]` attribute can be added directly to the field definition in the `#[pyclass]`.
 - For properties which require computation you can define `#[getter]` and `#[setter]` functions in the [`#[pymethods]`](#instance-methods) block.
```

Comment:
`#[getter]` and `#[setter]` probably do work with enums?

So I think maybe the "struct" clarification just needs to go on the first bullet, e.g. "simple struct fields", maybe.